### PR TITLE
updating to new dockerhub namespace

### PR DESCRIPTION
**Pull request by cskiraly**

### CODE REVIEW

**Comment by cskiraly** [comment]
Had no time to test it yet, but I want to make sure we are not using the old dockerhub namespace anymore. It could be a source of confusion ....

File: apps/agile-compose [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    ;;
+  x86_64 )
+    export IMAGE_INFLUXDB=influxdb
+    ;;
+  * )
+    export IMAGE_INFLUXDB=influxdb
+    ;;
+esac
+
+docker-compose $@ <<END
```
Comment (craig-mulligan) [inline apps/agile-compose]: Why pass the full file again? You can just run `docker compose` in the file with the docker-compose.yml

File: apps/agile-compose [hunk 2]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+export AGILE_CLIENT_PORT=2000
```
Comment (craig-mulligan) [inline apps/agile-compose]: I'd break the declaration of all envars into a separate file. This file should also get executed at build time (then if we add environment variable via the resin dashboard we can overwrite them) this makes cycles faster if you just want to change some config.

File: apps/docker-compose.yml [hunk 3]
```diff
@@ -21,3 +20,56 @@ services:
...
+      volumes:
+        - $HOME/.agile/nodered:/root/.node-red
+      ports:
+        - 1880:1880/tcp
+      links:
+        - agile-core:agile
+        - agile-influxdb:db
+      restart: always
+
+    agile-influxdb:
```
Comment (craig-mulligan) [inline apps/docker-compose.yml]: This is the data server right that I am building right or are you using influx for something else?

File: apps/agile [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```
Comment (craig-mulligan) [inline apps/agile]: where is this executed?

File: apps/agile [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  restart)
+    $0 stop
+    sleep 1
+    $0 start
+    ;;
+  update)
+    $BINDIR/agile-compose pull
+    ;;
+  *)
+    echo "Usage: $0 {start|stop|restart|update}"
```
Comment (craig-mulligan) [inline apps/agile]: Why not use the docker-compose API?

**Pull request closed without merging**